telegram-bot-bumblebee-core/src/main/resources/application.yml

```diff
@@ -56,3 +56,10 @@ spring:
     username:
     password:
     driver-class-name:
+
+autocomplete:
+    template:
+      - ДО СВИ ДОС
+      - ДОС ВИ ДОС
```

Comment:
Oh god, why not to store whole word and split it into 2/3 parts when we receive beginning of the phrase?